test.sh

```diff
@@ -96,17 +93,8 @@ function test_project {
   project_dir=$1
   version=$2
   header_text "performing tests in dir $project_dir for project version v$version"
-  vendor_tarball=$tmp_root/vendor.v$version.tgz
   old_gopath=$GOPATH
-  if [[ $version == "1" ]]; then
-      export GOPATH=$(pwd)/testdata/gopath
-      download_vendor_archive
-  fi
   cd testdata/$project_dir
-  # v2 uses modules, and thus doesn't have a vendor directory
-  [[ -e ${vendor_tarball} ]] && tar -zxf $vendor_tarball
-  make all test # v2 doesn't test on all by default
```

Comment:
Same as above comment, `make all test` needs to stay, the comment can go.

Reply:
Solved as well.